src/main/kotlin/gg/essential/vigilance/gui/settings/DropDownComponent.kt

```diff
@@ -85,8 +92,8 @@ class DropDownComponent(
         x = 2.pixels(alignOpposite = true)
```

Comment:
Another case of padding that's now handled by `scrollerContainer`.
```suggestion
        x = 0.pixels(alignOpposite = true)
```